src/lib/draw/EditGeometryMode.svelte

```diff
@@ -40,14 +40,16 @@
     name = interventionName(feature);
 
     if (feature.geometry.type == "LineString") {
-      $routeTool?.editExistingRoute(feature as Feature<LineString, Props>);
+      $routeTool?.editExistingRoute(
+        feature as unknown as Feature<LineString, RouteProps>,
+      );
       $routeTool?.addEventListenerSuccess(onSuccess);
       $routeTool?.addEventListenerUpdated(onUpdate);
       $routeTool?.addEventListenerFailure(onFailure);
       controls = "route";
     } else if (feature.geometry.type == "Polygon") {
       if (feature.properties.waypoints) {
-        $routeTool?.editExistingArea(feature as Feature<Polygon, Props>);
+        $routeTool?.editExistingArea(feature as Feature<Polygon, AreaProps>);
```

Comment:
We're not so precise about our properties in this repo's types, so the compiler gets confused because geometry type being polygon doesn't guarantee we have all the AreaProps